<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>tlinsWPilotHVSpeedDelta</class>
 <widget class="QWidget" name="tlinsWPilotHVSpeedDelta">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>542</width>
    <height>287</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Type</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QRadioButton" name="radioSteps">
        <property name="text">
         <string> Steps</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioDegris">
        <property name="text">
         <string> Degris</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Axis</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QRadioButton" name="radioHorisontal">
        <property name="text">
         <string> Horisontal</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioVertical">
        <property name="text">
         <string> Vertical</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupSteps">
     <property name="title">
      <string>Steps value</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="tlinsWDecimalValue" name="stepsValue" native="true"/>
      </item>
      <item>
       <widget class="QLabel" name="label">
        <property name="font">
         <font>
          <italic>true</italic>
         </font>
        </property>
        <property name="text">
         <string>[steps/s]</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupDegris">
     <property name="title">
      <string>Degris value</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="tlinsWCoordinateDegris" name="degrisValue" native="true"/>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="font">
         <font>
          <italic>true</italic>
         </font>
        </property>
        <property name="text">
         <string>[degris/s]</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>tlinsWCoordinateDegris</class>
   <extends>QWidget</extends>
   <header>tlinswcoordinatedegris.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>tlinsWDecimalValue</class>
   <extends>QWidget</extends>
   <header>tlinswdecimalvalue.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>radioDegris</sender>
   <signal>clicked()</signal>
   <receiver>tlinsWPilotHVSpeedDelta</receiver>
   <slot>degrisSelected()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>313</x>
     <y>56</y>
    </hint>
    <hint type="destinationlabel">
     <x>624</x>
     <y>104</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioSteps</sender>
   <signal>clicked()</signal>
   <receiver>tlinsWPilotHVSpeedDelta</receiver>
   <slot>stepsSelected()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>65</x>
     <y>56</y>
    </hint>
    <hint type="destinationlabel">
     <x>550</x>
     <y>172</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>degrisSelected()</slot>
  <slot>stepsSelected()</slot>
 </slots>
</ui>
